inst/htmljs/animint.js

```diff
@@ -993,8 +1059,20 @@ var animint = function (to_select, json_file) {
         });
         var chunk = nest.map(response);
         g_info.data[tsv_name] = chunk;
-        g_info.tr.select("td.downloaded").text(d3.keys(g_info.data).length);
         g_info.download_status[tsv_name] = "saved";
+        // Update size information after download
+        if(g_info.chunk_info && g_info.chunk_info[tsv_name]){
```

Comment:
this if block is repeated above for common_tsv.
please refactor by adding a function to avoid this repetition.